### DISCUSSION TITLE
[#969741] A cube appears in the world

### DISCUSSION THREAD

**Discussion by xKingDark** [Mojang/minecraft-editor#120] [Bugs]
A selection cube appears at the location ( X: 0 / Y: 0 / Z: 0 )

**How to Reproduce**:
  **1.** Create a flat world
  **2.** Go to ( X: 0 / Y: 0 / Z: 0 )

<img src="https://github.com/Mojang/minecraft-editor/assets/47607185/ee77f9f0-6309-4b21-8aec-bc5551546d4d">
<img src="https://github.com/Mojang/minecraft-editor/assets/47607185/e2f9a341-6b15-460b-8118-3af09f20f16f">
<img src="https://github.com/Mojang/minecraft-editor/assets/47607185/372367f2-e894-4b7e-826a-1eedce5dd5ca">

**Helpful Info**:
- Preview build version: `1.20.10.20`

**Comment by minecraftchloe** [comment]
Should be fixed with #130 !
